### PR TITLE
Make Header RTL-compatible 

### LINKED ISSUE #426: Make Header RTL-compatible
Action items:
- The navigation items should be aligned with the left side of the screen
- The dropdown items should not have a large leading gap
- Should the openverse logo be on the opposite side of the openverse brand text? cc @panchovm

### PR DESCRIPTION
**Pull request by dhruvkb**
## Fixes

Fixes #426 by @dhruvkb 

## Description

This PR
- fixes the dimensions of the Openverse logo by replacing `` with inline `<svg>`
- replaces hardcoded left/right with start/end either using Tailwind, (wherever possible) or right inside Bulma's CSS-mess
- adds RTL capability to `VIcon` and uses it for the external link icon
- aligns menu to the correct edge of the trigger
- makes some minor readability improvements and performs code cleanup

## Testing Instructions

See tracking issue #421 for instructions to reproduce. These instructions should no longer be able to reproduce the issue.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VIcon/VIcon.vue [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <template>
   <svg
     class="v-icon h-6 w-6"
+    :class="{ 'rtl-flip': rtlFlip }"
```
Comment (obulat) [inline src/components/VIcon/VIcon.vue:4]: Nice!

File: src/components/EmbeddedNavSection.vue [hunk 2]
```diff
@@ -1,40 +1,41 @@
 <template>
+  <!-- Refer to the Bulma docs for markup: https://bulma.io/documentation/components/navbar/ -->
   <nav :aria-label="$t('header.aria.primary')" class="navbar embedded">
     <NuxtLink to="/" style="align-self: center; line-height: 0">
-      <img
+      <!-- width and height chosen w.r.t. viewBox "0 0 280 42" -->
```
Comment (zackkrida) [inline src/components/EmbeddedNavSection.vue:5]: 👍

**Review by zackkrida — APPROVED** [review]
Looks great! The only thing I would add is a small bit of padding to the end of the logo when the lang is `rtl`:

> current appearance 

CleanShot 2021-11-18 at 10 49 34@2x

File: src/styles/components/navbar.sass [hunk 3]
```diff
@@ -82,11 +82,11 @@ $wp-max-width: 940px
   padding: 0.5rem 0.75rem
   position: relative
   .icon
-    margin-left: $space-small
+    margin-inline-start: $space-small
 
 .navbar.embedded
   .navbar-end
-    margin-left: auto
+    margin-inline-start: auto
```
Comment (obulat) [inline src/styles/components/navbar.sass:89]: I haven't even thought about converting the left-right values to inline-start and inline-end! Cool!

**Review by obulat — CHANGES_REQUESTED** [review]
Works great in RTL, the only thing (in addition to the Logo inline-start spacing) is the external icon on mobile: it is below the menu item instead of next to it:
Screen Shot 2021-11-19 at 2 01 11 PM

**Comment by dhruvkb** [comment]
@zackkrida 82f3d19 should fix the padding issues by updating markup as per Bulma docs.
@obulat 8ce88f5 should fix the icons that were incorrectly moving to newlines.

File: src/styles/components/navbar.sass (hunk 3 above)
Comment (obulat) [reply src/styles/components/navbar.sass:89]: There are still 4 occurrences of `-left` in this file, and one of them is causing the navbar burger to not be `inline-end`-aligned on mobile view in RTL. So, on RTL page the burger is next to Openverse logo.

**Review by obulat — CHANGES_REQUESTED** [review]
The icons work great, I will approve after the burger is end-aligned :)

File: src/styles/components/navbar.sass (hunk 3 above)
Comment (dhruvkb) [reply src/styles/components/navbar.sass:89]: I'm not able to reproduce this, the burger is positioned correctly for me:
Screenshot 2021-11-25 at 3 21 20 PM
Screenshot 2021-11-25 at 3 22 05 PM

Comment (obulat) [reply src/styles/components/navbar.sass:89]: I re-checked out the branch, and everything looks great, thank you!

**Review by obulat — APPROVED** [review]

**Pull request merged**